src/components/FilterSizes.js

```diff
@@ -8,58 +8,100 @@ class FilterSizes extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
+            paragraphs: [ //text, style.top, style.left
```

Comment:
Is there a reason that this isn't a constant? Do the paragraphs change?

Reply:
they don't change; i've moved it out of state